Rework README

TuringTux opened the pull request:
When I created a fork of this plugin, I also made some changes to the README to document my experiences with setting the plugin up. Now that all my code changes have been incorporated into this plugin, I'd like to discard my fork in an attempt to decrease entropy.

However, my README changes are not yet incorporated into this project, mostly because they are opinionated and also because they delete some content I found less relevant for my specific purpose.

I have opened this (draft) PR to evaluate the differences and then create a less-opinionated and complete version. After that, I would very much welcome your, @Monogramm's, feedback :)

TuringTux committed:
:memo: Rework README

TuringTux committed:
:rewind: Revert removal of badges

They weren't accurate for my fork, but are of course accurate for this repo

TuringTux committed:
:rewind: Remove superscript 2 from title

TuringTux committed:
:rewind: Revert fork description

TuringTux committed:
:fire: Remove comparison with Monogramm plugin

TuringTux committed:
:memo: Prepare installation w/o Docker section

madmath03 commented:
Thanks a lot @TuringTux for this proposal. I really like your section about how to setup this plugin with the official Taiga docker image.
I think we can add most of the original README's content to the "without docker section".

I will send some suggestions that you can apply / edit.

madmath03 committed:
:memo: Append the original README content

madmath03 commented on README.md:
````diff
@@ -4,14 +4,158 @@
+    LDAP_PORT = 389
+    LDAP_START_TLS = True
+    ```
+    What happens is that an unencrypted connection is established first, but then upgraded to a secure connection. To the best of my knowledge, this should also be safe – however, I like the `ldaps://` variant more.
````
Not exactly accurate: LDAPS will use a "LDAP SSL" connection, while enabling `LDAP_START_TLS ` will use a "LDAP over TLS" connection.
Both are secured, and in fact I believe TLS is recommended, but not all LDAP server support easily Start TLS so we provide both solutions.

TuringTux committed:
:memo: Tweak formatting

TuringTux committed:
:memo: Make formatting consistent

TuringTux committed:
:memo: Install extension from Git in Dockerfile

TuringTux committed:
:memo: Move back configuration to separate section

TuringTux committed:
:memo: Make headings consistent

TuringTux committed:
:memo: Reference conference section with link

TuringTux committed:
:memo: Add (redundant) information about target file

I hope that this makes the jump between sections a bit easier.

TuringTux committed:
:memo: Shorten config, merge further notes

The longer configuration that was initially in this README will be
re-added in the next commits.

TuringTux committed:
:memo: Separate paragraph

TuringTux committed:
:memo: Begin section "additional configuration options"

This re-adds (part of) the additional configuration from the original README

TuringTux committed:
:memo: Add Tls configuration

TuringTux committed:
:memo: Make message consistent

TuringTux committed:
:memo: Re-add last parts of original configuration

TuringTux commented on README.md:
````diff
@@ -4,14 +4,158 @@
+LDAP_MAP_USERNAME_TO_UID = None
+```
+
+Change the following fields matching your setup:
````
Done.

madmath03 commented:
Thanks @TuringTux for the support. PR merged.

TuringTux commented:
Thanks @madmath03 for merging :)
